proto/storage/rbac.proto

```diff
@@ -30,7 +30,7 @@ message K8sRole {
     map<string, string> annotations         = 8 [(gogoproto.moretags) = 'search:"Role Annotation"'];
     google.protobuf.Timestamp created_at    = 9;
 
-    repeated PolicyRule rules               = 10;
+    repeated PolicyRule rules               = 10 [(gogoproto.moretags) = 'sensorhash:"ignore"'];
```

Comment:
What happens if a rule is added or deleted?  would we miss those updates?  I'm admittedly not an expert on the hashing and deduping we do.

Reply:
No so it basically treats the slice as a set but all the values still must be tehre. The order just does not matter